pkgs/by-name/kc/kcl-language-server/package.nix

```
@@ -0,0 +1,29 @@
+{
+  lib,
+  rustPlatform,
+  fetchFromGitHub,
+  pkg-config,
+}:
+rustPlatform.buildRustPackage rec {
```

```suggestion
}:

rustPlatform.buildRustPackage rec {
```

Would you mind adding a space here?

---

done

---

ok not done yet :)